### PR TITLE
Fixed a, blockquote, strong tags in darkmode

### LINKED ISSUE #5922: [DOCS] link color is camouflage with background in docs
### Description

Link color is camouflage with background 

Visit: https://linkfree.io/docs/environments/environment-variables and scroll down to bottom

I am able to see because is selected.
Screenshot 2023-04-08 at 2 34 11 PM

If not selected: 
Screenshot 2023-04-08 at 2 36 36 PM

Screenshot 2023-04-08 at 2 37 08 PM

### Screenshots

_No response_

### Additional information

_No response_

### LINKED ISSUE #5926: [DOCS] blockquote text is not visible in docs/hacktoberfest
### Description

**Blockquote** text color is not visible.

Visit : https://linkfree.io/docs/contributing/hacktoberfest

### Screenshots

This is the screenshot without selecting text.

image

After selecting text.

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ChinmayMhatre**
## Fixes Issue
closes #5922 closes #5926 

image

image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6000"></a>

### CODE REVIEW

**Comment by ChinmayMhatre** [comment]
A good first issue can be moving the li, table tag to separate components similar to these tags

File: components/mdx/BlockQuote.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const BlockQuote = ({children}) => {
+    return (
+        <blockquote className='text-primary-medium dark:text-primary-low-medium'>{children}</blockquote>
```
Comment (eddiejaoude) [inline components/mdx/BlockQuote.js:5]: I think should be double quotes

File: components/mdx/ComponentStyle.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
+import Link from "@components/Link";
+import Strong from "@components/mdx/Strong";
+import BlockQuote from "@components/mdx/BlockQuote";
```
Comment (eddiejaoude) [inline components/mdx/ComponentStyle.js:3]: Blank link between import and export would be more readable and match other files

File: components/mdx/Strong.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const Strong = ({children}) => {
+    return (
```
Comment (eddiejaoude) [inline components/mdx/Strong.js:2]: I think indentation might not math our style

File: components/mdx/Strong.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+        <span className=" font-bold">{children}</span>
```
Comment (eddiejaoude) [inline components/mdx/Strong.js:3]: I don't think the extra space is needed here

```suggestion
        <span className="font-bold">{children}</span>
```

File: components/mdx/Strong.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    )
+}
+
+export default Strong
```
Comment (eddiejaoude) [inline components/mdx/Strong.js:7]: Ending blank line is recommended by GitHub

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
@ChinmayMhatre looks great 👍  thank you for making these contributions 

Just some formatting feedback

**Review by eddiejaoude — APPROVED** [review]
These are super important improvements, so I am keen to get them in, I will fix the formatting via a temporary branch (I hope you don't mind)

File: components/mdx/BlockQuote.js [hunk 6]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
[the remark this answers is not in the document]
Comment (eddiejaoude) [reply components/mdx/BlockQuote.js:1]: Actually, I don't think this import is used

**Comment by eddiejaoude** [comment]
no problem Chinmay, you did all the hard work 💪 , I am happy to reformat because that is the easy part

**Comment by eddiejaoude** [comment]
yes your PR has been merged, reformatted and deployed 👍

**Comment by ChinmayMhatre** [comment]
> yes your PR has been merged, reformatted and deployed 👍

Oh no I was asking if. Like we have separate components for a, blockquotes, strong can we do it for table, li in the `componentStyle` file?
image

**Comment by ChinmayMhatre** [comment]
> Sorry @ChinmayMhatre , I read your previous comment too fast and missed your question 🤦‍♂️
> 
> Yes we should do this, if you like can you create an issue for each one, and maybe those could be `good first issue` for someone else to do

I think we should make it a `good first issue` it can be a good one for first timers 💪🏻. Should I create separate issue for `table` and `li`? Wouldn't it create a conflict? it'd be confusing for beginners? what do you think?

**Comment by eddiejaoude** [comment]
I think make them separate issues, I can always resolve conflicts if any - although they are different lines so hopefully git figures it out ok - that way 2 people can get started with contributing

**Pull request merged**